docs/usage/the-litestar-app.rst

```diff
@@ -68,6 +68,20 @@ establish the connection, and another to close it, and then pass them to the Lit
     :language: python
 
 
+Lifespan context managers
+-------------------------
+
+In addition to the lifespan hooks, Litestar also supports managing the lifespan of an
+application using asynchronous context managers. This can be useful when dealing with
```

Comment:
```suggestion
application using `asynchronous context managers <https://bbc.github.io/cloudfit-public-docs/asyncio/asyncio-part-3.html>`_. This can be useful when dealing with
```

Reply:
This was a nice read around that, if we want to include it

Reply:
While a good read indeed, I don't think this belongs here. If we want we can link to the stdlib about those, but since this isn't a tutorial but rather a brief explanation, we assume the reader is already familiar with the topics being discussed.